server/repository/src/db_diesel/filter_sort_pagination.rs

```diff
@@ -101,16 +102,22 @@ impl StringFilter {
     }
 }
 
-#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+#[derive(Clone, PartialEq, Debug, TS, Serialize, Deserialize)]
 pub struct EqualFilter<T>
 where
     T: 'static,
 {
+    #[ts(optional)]
```

Comment:
Without optional Option<T> is treated as null, with optional treated as undefined